.circleci/config.yml

```diff
@@ -69,109 +57,103 @@ jobs:
+  integration-test_deploy:
     when: << pipeline.parameters.run-integration-tests >>
     jobs:
-      - integration-test-install-v2:
+      # Run any integration tests defined within the `jobs` key.
+      - integration-test-install:
           matrix:
             parameters:
-              executor: [aws-cli/default, without-less, macos, machine, python, awsv2preinstalled]
-      - integration-test-install-v1:
-          matrix:
-            parameters:
-              executor: [macos, machine, python]
-      - integration-test-skip-install:
-          matrix:
-            parameters:
-              executor: [macos, machine, python]
-      - replace-pip-aws-test
-      # This job is supposed to fail, do not run it unless for testing purposes.
-      # This job fails with a more useful error message when a user wants to install CLI V1 on an image without python.
-      # - integration-test-install-v1:
-      #     name: v1-docker
-      #     executor: without-less
-
+              executor: ["docker-base", "macos", "arm"]
+          post-steps:
+            - check_aws_version
+      - integration-test-install:
+          name: integration-test-install-version
+          executor: docker-base
+          version: "2.1.10"
+          post-steps:
+            - check_aws_version:
+                version: "2.1.10"
+      - integration-test-install:
+          name: integration-test-install-override-version
+          executor: docker-base
+          version: "2.1.10"
+          post-steps:
+            - check_aws_version:
+                version: "2.1.10"
+            - aws-cli/setup:
+                version: "2.0.20"
+                override-installed: true
+            - check_aws_version:
+                version: "2.0.20"
+      # Publish a semver version of the orb. relies on
+      # the commit subject containing the text "[semver:patch|minor|major|skip]"
+      # as that will determine whether a patch, minor or major
+      # version will be published or if publishing should
+      # be skipped.
+      # e.g. [semver:patch] will cause a patch version to be published.
       - orb-tools/dev-promote-prod-from-commit-subject:
           orb-name: circleci/aws-cli
-          context: orb-publishing
-          add-pr-comment: true
-          bot-token-variable: GHI_TOKEN
-          bot-user: cpe-bot
+          context: orb-publisher
+          add-pr-comment: false
           fail-if-semver-not-indicated: true
-          publish-version-tag: true
-          ssh-fingerprints: ed:dc:9c:fd:ab:7e:16:5b:28:9f:98:68:5f:80:82:dd
+          publish-version-tag: false
+          requires:
+            - integration-test-install
+            - integration-test-install-version
+            - integration-test-install-override-version
           filters:
             branches:
-              only: master
-          requires: *integration_tests
+              only:
+                - master
+                - main
```

Comment:
Why is both main and master here?

Reply:
This is the default for the ODK but to your point, not relevant here. As we use the `master` branch on this project, I will remove "main"